app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Proxying with {{site.base_gateway}}
```

Comment:
For reviewers: Is this too much info for the proxying page? Should proxying just contain info about proxying and the connective parts (ex. listeners, routing, plugins, etc) should go on a traffic control landing page with proxying as another card/section instead? This is similar to what we had in the offsite proxy branch: https://github.com/Kong/developer.konghq.com/compare/main...proxy-reference#diff-8922116eb8441698da97efe2beeb185fb024118f8add77981f8eeb930867bbc6

I thought listeners, etc. could be helpful on this page to understanding proxying, but wasn't sure if others felt it was out of place.